tests/tr1/tests/cmath1/test.cpp

```diff
@@ -8,6 +8,8 @@
 #include "tdefs.h"
 #include <cmath>
 
+#define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)

```

Comment:
I kind of wish there was a unified place `STATIC_ASSERT` could be defined. Not the end of the world though.

Reply:
I searched for `STATIC_ASSERT` and found that it is defined in many source files, but all those previous definitions are in std tests.